tutorials/algorithms/10_pvqd.ipynb

```diff
@@ -0,0 +1,591 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Projected Variational Quantum Dynamics\n",
+    "\n",
+    "The projected Variational Quantum Dynamics (p-VQD) algorithm is a quantum algorithm for real time evolution. It's a variational algorithm that projects the state at time $t + \\Delta_t$, as calculated with Trotterization, onto a parameterized quantum circuit. \n",
+    "\n",
+    "For a quantum state $|\\phi(\\theta)\\rangle = U(\\theta)|0\\rangle$ constructed by a parameterized quantum circuit $U(\\theta)$ and a Hamiltonian $H$, the update rule can be written as\n",
+    "\n",
+    "$$\n",
+    "\\theta_{n+1} = \\theta_n + \\arg\\min_{\\delta\\theta} 1 - |\\langle\\phi(\\theta_n + \\delta\\theta)|e^{-i\\Delta_t H}|\\phi(\\theta_n)\\rangle|^2,\n",
+    "$$\n",
+    "\n",
+    "where $e^{-i\\Delta_t H}$ is calculated with a Trotter expansion (using e.g. the `PauliEvolutionGate` in Qiskit!).\n",
+    "\n",
+    "The following tutorial explores the p-VQD algorithm in Qiskit, which is implemented available as `qiskit.algorithms.time_evolvers.PVQD`. For details on the algorithm, see the original paper: [Barison et al. Quantum 5, 512 (2021)](https://quantum-journal.org/papers/q-2021-07-28-512/#)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The example we're looking at is the time evolution of the $|00\\rangle$ state under the Hamiltonian\n",
+    "\n",
+    "$$\n",
+    "    H = 0.1 Z_1 Z_2 + X_1 + X_2,\n",
+    "$$\n",
+    "\n",
+    "which is an Ising Hamiltonian on two neighboring spins, up to a time $T=1$ and we want to keep track of the total magnetization $M = Z_1 + Z_2$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "final_time = 1\n",
+    "hamiltonian = SparsePauliOp.from_sparse_list([\n",
+    "    (\"ZZ\", [0, 1], 0.1), (\"X\", [0], 1), (\"X\", [1], 1),\n",
+    "], num_qubits=2)\n",
+    "observable = SparsePauliOp([\"ZI\", \"IZ\"])"
```

Comment:
I think it's more readable if you explicitly include `magnetization` in the variable name, maybe `magnetization_observable`? (I know it's a bit long, open to other ideas)